src/features/App/WorkspaceManager/ProfileCreator/index.tsx

```diff
@@ -80,14 +93,23 @@ export const ProfileCreator: FC<ProfileCreatorProps> = ({
 					>
 						Create profile
 					</Button>
-					<Button
-						variant="secondary"
-						w="100%"
-						onClick={() => onPressCreate(false)}
-						disabled={isPending}
-					>
-						Continue with no password
-					</Button>
+					<VStack w="100%" gap={'0rem'}>
+						<Button
+							variant="secondary"
+							w="100%"
+							onClick={() => onPressCreate(false)}
+							disabled={isPending}
+						>
+							Continue with no encryption
+						</Button>
+						<Text
+							alignSelf={'start'}
+							color={'typography.additional'}
+							fontSize={'14px'}
+						>
+							Insecure: your data will not be encrypted
```

Comment:
Remove it and show modal window with confirmation when user click option with no encryption